custom/succeed/reports/all_patients.py

```diff
@@ -334,6 +356,37 @@ def es_results(self):
                 }
             }
             q['sort'] = sort
+        elif sorting_block == 'care_team':
+            ids_and_names = {}
+            user_ids = {}
+
+            users = []
+            for g in Group.by_domain(SUCCEED_DOMAIN):
```

Comment:
It could be worth caching this since they probably aren't going to add new groups often.